scripts/actions/remove-completed-batch.js

```diff
@@ -107,4 +109,4 @@ const removePageContext = async (fileUris) => {
   }, 'SUCCESS');
 };
 
-setInProgressToDone();
+setInProgressToDone(PROJECT_ID);
```

Comment:
i made this an argument just for it to be clearer that this runs for a given project. 

Having it be part of the signature makes it explicit versus just having it be internal to the method.